gti.c

```diff
@@ -71,17 +72,20 @@ int main(int argc, char **argv)
 {
     int i;
     char *git_path;
-    (void) argc;
+    int pushing = check_push_command(argc, argv);
 
     open_term();
     TERM_WIDTH = term_width();
     SLEEP_DELAY = 1000000 / (TERM_WIDTH + GTI_SPEED);
 
     init_space();
     for (i = -20; i < TERM_WIDTH; i++) {
+      if (pushing)
+        push_car(i);
+      else
         draw_car(i);
-        usleep(SLEEP_DELAY);
-        clear_car(i);
+      usleep(SLEEP_DELAY);
+      clear_car(i);
```

Comment:
This is still using two spaces ;) but I'll add an `indent` script or somthing to the Makefile. Okay for now.